src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.Aapt2.targets

```
@@ -105,22 +105,35 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
   </ItemGroup>
 </Target>
 
-<Target Name="_CompileResources"
+<Target Name="_ConvertResourcesCases"
     Condition=" '$(_AndroidUseAapt2)' == 'True' "
     Inputs="@(AndroidResource)"
-    Outputs="$(_AndroidLibraryFlatArchivesDirectory)\compiled.stamp"
+    Outputs="$(_AndroidStampDirectory)_ConvertResourcesCases.stamp"
+    DependsOnTargets="$(_BeforeConvertResourcesCases)"
   >
   <MakeDir Directories="$(_AndroidLibraryFlatArchivesDirectory)" Condition="!Exists('$(_AndroidLibraryFlatArchivesDirectory)')" />
   <!-- Change cases so we support mixed case resource names -->
   <ConvertResourcesCases
       ContinueOnError="$(DesignTimeBuild)"
       AcwMapFile="$(_AcwMapFile)"
       AdditionalResourceDirectories="@(_LibraryResourceDirectories)"
-      AndroidConversionFlagFile="$(_AndroidLibraryFlatArchivesDirectory)\compiled.flata"
+      AndroidConversionFlagFile="$(_AndroidStampDirectory)_ConvertResourcesCases.stamp"
       CustomViewMapFile="$(_CustomViewMapFile)"
       ResourceDirectories="$(MonoAndroidResDirIntermediate)"
       ResourceNameCaseMap="$(_AndroidResourceNameCaseMap)"
   />
+  <Touch Files="$(_AndroidStampDirectory)_ConvertResourcesCases.stamp" AlwaysCreate="True" />
+  <ItemGroup>
+    <FileWrites Include="$(_AndroidStampDirectory)_ConvertResourcesCases.stamp" />
+  </ItemGroup>
+</Target>
+
+<Target Name="_CompileResources"
+    Condition=" '$(_AndroidUseAapt2)' == 'True' "
+    Inputs="@(AndroidResource)"
+    Outputs="$(_AndroidLibraryFlatArchivesDirectory)\compiled.stamp"
```

Since we are changing this target anyway, can it follow the new convention?
```suggestion
    Outputs="$(_AndroidStampDirectory)_CompileResources.stamp"
```
Then there might also be a `<FileWrites/>` you can remove for it, too.